spec/unit/cli_spec.rb

```diff
@@ -252,7 +265,7 @@
               an_instance_of(CurrentUser),
               Inspector.all_scopes,
               an_instance_of(Filter),
-              {}
+              { :remote_user => "root" }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.